src/persist/EpochSys.hpp

```diff
@@ -196,6 +221,22 @@ class EpochSys{
 
     static void init_thread(int _tid){
         EpochSys::tid = _tid;
+        Ralloc::set_tid(_tid);
```

Comment:
No, those who don't call Ralloc methods don't have to call `set_tid()`. Do these threads call `init_thread()`? I thought this func is called only in worker threads (and also epoch advancer since it deallocates things)